css/small.css

```diff
@@ -0,0 +1,159 @@
+/* FOR SMALLER SCREENS <768PX */
+
+@media only screen and (max-width: 768px) {
+    .featured,
+    .latest {
+        width: 100%;
+        justify-content: center;
+    }
+    .lead-article {
+        grid-column-start: 1;
+        grid-column-end: 3;
+        grid-row-start: 1;
+        grid-row-end: 3;
+        background-image: url('/images/cover.png');
+        height: auto;
+        background-size: cover;
+        color: white;
+        display: inline-block;
+        text-align: start;
+        background-repeat: no-repeat;
+        height: 125px;
+    }
+    .funding-article {
+        height: fit-content;
+    }
+    .latest-article {
+        display: grid;
+        grid-template-columns: 1fr 7fr;
+    }
+    .latest-article>img {
+        width: 60px;
+        height: 60px;
+        grid-row-start: 1;
+        grid-row-end: 3;
+        padding: 5px;
+    }
+    .latest-article>a {
+        display: inline-block;
+        text-decoration: none;
+        padding: 1px;
+    }
+    .latest-article>a>h3 {
+        color: #000;
+        font-size: 0.9em;
+        font-weight: 700;
+    }
+    .latest-article>a>h5 {
+        color: #131313;
+        font-size: 0.7em;
+    }
+    .links {
+        display: none;
+    }
+    header {
+        width: 100%;
+        position: sticky;
+        top: 0;
+    }
+    nav {
+        display: flex;
+        align-items: center;
+        flex-direction: row;
+        border: 2px solid brown;
```

Comment:
Why do we have a red border here? :thinking:

Reply:
Forgot to remove it, I was testing the layout before adding content